packages/aws-client/package.json

```diff
@@ -43,6 +43,7 @@
   "author": "Cumulus Authors",
   "license": "Apache-2.0",
   "dependencies": {
+    "@aws-sdk/types": "^3.58.0",
```

Comment:
Fix the issue for release ci build. The @aws-sdk/types@3.289.0 doesn't exist.  From npm log, it seems to be related to @aws-sdk/abort-controller@3.289.0 which is just publish an hour ago.

lerna ERR! npm install --ignore-scripts --no-package-lock exited 1 in '@cumulus/aws-client'
error	10-Mar-2023 15:23:33	lerna ERR! npm install --ignore-scripts --no-package-lock stderr:
error	10-Mar-2023 15:23:33	npm ERR! code ETARGET
error	10-Mar-2023 15:23:33	npm ERR! notarget No matching version found for @aws-sdk/types@3.289.0.
error	10-Mar-2023 15:23:33	npm ERR! notarget In most cases you or one of your dependencies are requesting
error	10-Mar-2023 15:23:33	npm ERR! notarget a package version that doesn't exist.